<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="fav_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_all">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить все</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
